apps/dashboard/test/test_helper.rb

```diff
@@ -55,6 +57,11 @@ def teardown_usr_fixtures
   def bc_ele_id(ele)
     "batch_connect_session_context_#{ele}"
   end
+
+  def button_link(text, link)
```

Comment:
Put this suggestion here, but you'd have to change it in other places.

```suggestion
  def button_link?(text, link)
```